src/upgrade/upgrade_scripts/5.3.0/update_bucket_owner_account.js

```diff
@@ -1,31 +0,0 @@
-/* Copyright (C) 2016 NooBaa */
-"use strict";
-
-const _ = require('lodash');
-
```

Comment:
what's the motivation behind removing old upgrade scripts??

Reply:
I agree with @romayalon 
I don't think we should remove those. 
although they are old, they are usually used as a reference to writing new ones. 
@nadavMiz  Please revert the removal of all the upgrade scripts.